apix_documentation_swagger_v4.yaml

```diff
@@ -0,0 +1,1186 @@
+                      description: "The type of the image: url or uri"
+                    value:
+                      type: string
+                      description: The URL of the image for url or the content
+                        of the image for uri
+                      example: http://via.placeholder.com/350x150
+
+    MFA_Challenge_Image_Choices:
+      description: Structure of an MFA Challenge of type 'image_choice'
+      allOf:
+        - $ref: '#/components/schemas/GenericData'
+      type: object
+      required:
+        - attributes
+      properties:
+        attributes:
+          type: object
+          allOf:
+            - $ref: '#/components/schemas/MFA_Challenge'
+            - type: object
+              required:
+                - prompt
+              properties:
+                prompt:
+                  type: array
+                  description: An array of images
+                  items:
+                    type: object
+                    properties:
+                      id:
+                        type: integer
+                        description: The ID of the option (basically the index
+                          of the array)
+                      type:
+                        type: string
+                        description: "The type of the image: url or uri"
+                      value:
+                        type: string
+                        description: The URL of the image for url or the content
+                          of the image for uri
+                        example: http://via.placeholder.com/350x150
+
+    MFA_Challenge_Choices:
+      description: Structure of an MFA Challenge of type '?'
+      allOf:
+        - $ref: '#/components/schemas/GenericData'
+      type: object
+      required:
+        - attributes
+      properties:
+        attributes:
+          type: object
+          allOf:
+            - $ref: '#/components/schemas/MFA_Challenge'
+            - type: object
+              required:
+                - prompt
+              properties:
+                prompt:
+                  type: array
+                  description: An array of choices
+                  items:
+                    type: object
+                    properties:
+                      id:
+                        type: integer
+                        description: The ID of the option (basically the index
+                          of the array)
+                      category:
+                        type: string
+                        description: Defines the category to which the value
+                          belongs (e.g. Phone, Text, E-mail). It can be null
+                        example: Phone
+                      value:
+                        type: string
+                        description: A phone number or e-mail address. The same
+                          value can be present in different categories
+                        example: "+1234567890"
```

Comment:
Same comments for this challenge

Reply:
@jdsampayo I think this is not yet implemented in Plunderer::Faker::Client, right?